website/docs/gen-ai/inference/nvidia-nim-llama3.md

```diff
@@ -0,0 +1,340 @@
+  tag: latest
+```
+
+**4. Verify the deployed model**
+
+Once all pods in `nim` namespace is ready with `1/1` status, use below command to verify it's ready to serve the traffic.
+
+```bash
+export INGRESS_URL=$(kubectl get ingress -n nim -o jsonpath='{.items[0].status.loadBalancer.ingress[0].hostname}')
+
+curl -X 'POST' \
+"http://$INGRESS_URL/v1/completions" \
+-H 'accept: application/json' \
+-H 'Content-Type: application/json' \
+-d '{
+"model": "meta/llama3-8b-instruct",
+"prompt": "Once upon a time",
+"max_tokens": 64
+}'
+```
+
+you will see similar output like the following
+
+```json
+{"id":"cmpl-63a0b66aeda1440c8b6ca1ce3583b173","object":"text_completion","created":1719742336,"model":"meta/llama3-8b-instruct","choices":[{"index":0,"text":", there was a young man named Jack who lived in a small village at the foot of a vast and ancient forest. Jack was a curious and adventurous soul, always eager to explore the world beyond his village. One day, he decided to venture into the forest, hoping to discover its secrets.\nAs he wandered deeper into","logprobs":null,"finish_reason":"length","stop_reason":null}],"usage":{"prompt_tokens":5,"total_tokens":69,"completion_tokens":64}}
+```
+
+### Testing the Llama3 model deployed with NIM
+It's time to test the Llama3 just deployed. We will run the following commands with the same prompts to verify the generated outputs.
+
+First, expose the model serving service with port-forward using kubectl
+
+```bash
+kubectl port-forward -n nim svc/nim-llm 8000
+```
+
+Next, open another Terminal window, run the client using the existing prompts:
+
+```bash
+cd gen-ai/inference/nvidia-nim/nim-client
+python3 client.py --input-prompts prompts.txt --results-file results.txt
+```
+
+You will see an output something like below:
+
+```text
+Loading inputs from `prompts.txt`...
+Model meta/llama3-8b-instruct - Request 14: 4655.84 ms
+Model meta/llama3-8b-instruct - Request 10: 6463.45 ms
+Model meta/llama3-8b-instruct - Request 3: 7787.67 ms
+Model meta/llama3-8b-instruct - Request 15: 7794.37 ms
+Model meta/llama3-8b-instruct - Request 1: 8320.75 ms
+Model meta/llama3-8b-instruct - Request 12: 8925.80 ms
+Model meta/llama3-8b-instruct - Request 5: 9079.14 ms
+Model meta/llama3-8b-instruct - Request 16: 9501.56 ms
+Model meta/llama3-8b-instruct - Request 18: 9790.65 ms
+Model meta/llama3-8b-instruct - Request 0: 10248.90 ms
+Model meta/llama3-8b-instruct - Request 6: 10243.54 ms
+Model meta/llama3-8b-instruct - Request 4: 10596.04 ms
+Model meta/llama3-8b-instruct - Request 11: 10798.81 ms
+Model meta/llama3-8b-instruct - Request 17: 10793.39 ms
+Model meta/llama3-8b-instruct - Request 8: 10977.41 ms
+Model meta/llama3-8b-instruct - Request 2: 12085.08 ms
+Model meta/llama3-8b-instruct - Request 19: 12645.12 ms
+Model meta/llama3-8b-instruct - Request 9: 13346.84 ms
+Model meta/llama3-8b-instruct - Request 13: 13581.21 ms
+Model meta/llama3-8b-instruct - Request 7: 14917.04 ms
```

Comment:
print seconds as well

Reply:
done. keep same format as other place (seconds and ms)